File: COM_SIC_Activaciones/DataPowerRest/HeadersRequest.vb
'INC000002161718 inicio
Imports System

Public Class HeadersRequest

    Private _HeaderRequest As HeaderRequest

    Public Property HeaderRequest() As HeaderRequest
        Get
            Return _HeaderRequest
        End Get
        Set(ByVal Value As HeaderRequest)
            _HeaderRequest = Value
        End Set
    End Property

    Public Sub New()
        _HeaderRequest = New HeaderRequest
    End Sub

End Class

File: COM_SIC_Procesa_Pagos/DataPowerRest/HeadersRequest.cs
using System;

namespace COM_SIC_Procesa_Pagos.DataPowerRest
{
	public class HeadersRequest
	{
		private HeaderRequest _headerRequest ;
		
		public HeaderRequest HeaderRequest
		{
			set{_headerRequest= value;}
			get{ return _headerRequest;}
		}

		public HeadersRequest()
		{
		_headerRequest= new HeaderRequest();
		}
	}
}
